pkg/operator/cluster/spec.go

```diff
@@ -78,3 +78,10 @@ func (p PlacementSpec) GetMDS() k8sutil.Placement { return p.All.Merge(p.MDS) }
 func (p PlacementSpec) GetMON() k8sutil.Placement { return p.All.Merge(p.MON) }
 func (p PlacementSpec) GetOSD() k8sutil.Placement { return p.All.Merge(p.OSD) }
 func (p PlacementSpec) GetRGW() k8sutil.Placement { return p.All.Merge(p.RGW) }
+
+// TODO: Pull in the spec from the kubernetes tree
+type VolumeAttachSpec struct {
```

Comment:
help me understand why this is needed in master right now before the in-tree volume plug in is available?